src/pyOpenMS/pxds/MultiplexDeltaMassesGenerator.pxd

```diff
@@ -13,8 +13,8 @@ cdef extern from "<OpenMS/TRANSFORMATIONS/FEATUREFINDER/MultiplexDeltaMassesGene
         MultiplexDeltaMassesGenerator(String labels, int missed_cleavages, libcpp_map[ String, double ] label_mass_shift) nogil except +
 
         void generateKnockoutDeltaMasses() nogil except +
-        void printSamplesLabelsList() nogil except +
-        void printDeltaMassesList() nogil except +
+        void printSamplesLabelsList(std::ostream &stream) nogil except +
```

Comment:
@hroest Could you please comment? I am unsure how to fix the `pxd`. Thanks.

Reply:
I think this cannot be wrapped in python (but it should be trivial to just print the array in python).

Reply:
So I remove both methods from the wrapper for now?